jekyll/_cci2/hello-world.adoc

```diff
@@ -0,0 +1,531 @@
+---
+redirect_from: /examples-intro/
+tags:
+  platform:
+    - Cloud
+    - Server v4.x
+    - Server v3.x
+    - Server v2.x
+sectionTags:
+  Cloud:
+    - "#echo-hello-world-cloud"
+  Server_v4:
+    - "#echo-hello-world-server-4"
+  Server_v3:
+    - "#echo-hello-world-server-3"
+  Server_v2:
+    - "#echo-hello-world-server-2"
+---
+= Hello world
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Get started with CircleCI by printing Hello World in any execution environment.
+:icons: font
+
+This page describes how to get started with a basic pipeline using any execution environment. The steps described here assume you want to create a new project locally and push your changes to your VCS using `git`. 
+
+As an alternative, if you already have a repository you want to "plug in" to CircleCI you can set this up entirely within the CircleCI web app, including commiting a config file. This flow is outlined in the link:/docs/getting-started[Quickstart guide].
```

Comment:
I think this paragraph could be moved to after the next one (i.e. "This page assumes you have already signed up for CIrcleCI)